File: ProgramingBasics/Simple-Loops/08.OddEvenSum/src/OddEvenSum.java
import java.util.Scanner;

public class OddEvenSum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        int sumOdd = 0;
        int sumEven = 0;

        for (int i = 0; i < n; i++) {
            int num = Integer.parseInt(scanner.nextLine());

            if (i % 2 == 0) {
                sumEven += num;
            }
            else if (i % 2 != 0) {
                sumOdd += num;
            }
        }

        int diff = Math.abs(sumEven - sumOdd);

        if (sumEven == sumOdd) {
            System.out.println("Yes");
            System.out.println("Sum = " + sumEven);
        }
        else {
            System.out.println("No");
            System.out.println("Diff " + diff);
        }
    }
}

File: ProgramingBasics/SimpleLoops/08.OddEvenSum/08.OddEvenSum/OddEvenSum.cs
using System;

namespace _08.OddEvenSum
{
    public class OddEvenSum
    {
        public static void Main(string[] args)
        {
            var n = int.Parse(Console.ReadLine());
            var sumOdd = 0;
            var sumEven = 0;

            for (var i = 0; i < n; i++)
            {
                int num = int.Parse(Console.ReadLine());

                if (i % 2 == 0)
                {
                    sumEven += num;
                }
                else if (i % 2 != 0)
                {
                    sumOdd += num;
                }
            }

            var diff = Math.Abs(sumEven - sumOdd);

            if (sumEven == sumOdd)
            {
                Console.WriteLine("Yes");
                Console.WriteLine($"Sum = {sumEven}");
            }
            else
            {
                Console.WriteLine("No");
                Console.WriteLine($"Diff {diff}");
            }
        }
    }
}
